### ISSUE TITLE
Allow different ports for the server/client

### ISSUE THREAD

**Issue by shuther** [DropbaseHQ/dropbase#17]
I need to adapt the docker-compose.yml files to bind to different ports; for example
```yml
    ports:
      - "3618:80"
```

Could you confirm where to change it in the code so the client can connect to the server on a different port? same question related to lsp and redis?

**Comment by azhakhan** [comment]
yes, you can use different port for the client by updating docker-compose.
but you'd also need to add cors_origins record to server.toml, as
`cors_origins = ["http://localhost:3618"]`

make sure to add it before your llm creds

you should be able to map redis to another port as well via docker-compose, just make sure to add this line to server.toml
`redis_host = "redis"`
that way, the server will resolve redis by container name ("redis")

as for the server and lsp, it's not possible use different ports yet. we'll update the client and notify you once it's possible

**Comment by kiranraoboinapally** [comment]
@RestController
public class MyController {
	@CrossOrigin(origins = "http://localhost:your_portnumber")
	 @GetMapping("/hello")
	    public String sayHello() {
	        return "Hello from Spring Boot!";
	 }
}

Like this you can do it as example
